test/quantization/fx/test_quantize_fx.py

```diff
@@ -6002,6 +6003,8 @@ def forward(self, x):
                 # F.tanh is deprecated
                 x = torch.tanh(x)
                 x = x.tanh()
+                # TODO(future PR): handle F.softmax
```

Comment:
we can wait a bit and support in the new api: https://github.com/pytorch/pytorch/pull/75135

Reply:
yeah I'm talking about new additions, not this one, please feel free to land this for urgent use cases